lbrynet/core/client/ConnectionManager.py

```diff
@@ -142,37 +142,21 @@ def _send_secondary_requests(self, peer, protocol):
         ]
         return defer.DeferredList(ds)
 
```

Comment:
protocol_disconnected() renamed to _peer_disconnected(), and does not require protocol as argument. ( only used for debugging here, and its always going to be ClientProtocol anways)